lib/myFTClient.js

```diff
@@ -436,6 +436,27 @@ function _getNodesFollowingConcept(licenceId, conceptId, nodeType) {
 }
 
 /**
+ * Gets nodes that are following concepts/topics through KAT
+ * @param {String} licenceId -
+ * @param {String} conceptId -
+ * @param {String} nodeType - user|group
+ * @param {Object} params - Any additional parameters needed to filter the follow relationship.
+ * @returns {Promise} response -
+ * @private
+ */
+function _getNodesFollowingConceptByKat (licenceId, conceptId, nodeType, params) {
+  const operation = `myFTClient.getNodesFollowingConceptByKat`;
+  log.debug({operation, licenceId, conceptId, nodeType, params});
+
+  return _createAndTriggerScopedRequestKat('GET', myftConst.licenceNodeName, licenceId, myftConst.topicNodeName, conceptId, myftConst.followedRelName, nodeType, null, params)
```

Comment:
`myftConst.followedRelName` here is actually `followed` but the route requires this part to be `followers`. You can maybe hardcode this here as it's the only route that uses this word or we can change it in the `next-myft-api` to use `followed`.